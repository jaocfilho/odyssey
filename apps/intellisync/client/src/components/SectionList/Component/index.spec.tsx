import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import { SectionListComponent } from '.';

describe('SectionListComponent', () => {
  it('should render without crashing', () => {
    const { container } = render(
      <SectionListComponent>children</SectionListComponent>
    );

    expect(container).toMatchSnapshot();
  });
});
