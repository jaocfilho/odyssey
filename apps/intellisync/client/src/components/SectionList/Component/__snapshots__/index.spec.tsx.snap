// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SectionListComponent > should render without crashing 1`] = `
<div>
  <div
    class="divide-y dark:divide-zinc-50/10"
  >
    children
  </div>
</div>
`;
